### DISCUSSION TITLE
[Feature] Client-js error and user association

### DISCUSSION THREAD

**Discussion by gamlungJoeng** [apache/skywalking#10313] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

For web pages, errors may be related to user data. For example, dirty data may only exist for certain special users. It would be very helpful when fixing these bugs if you could document the relationship between the bug and the user

### Use case

When I see an error in the UI console, I can also see which users have this error, which will speed up my fixes

### Related issues

Whether the custom Tag can be displayed in the error and pass through each node (web -> oap -> metrics -> alarm -> skywalkingUI)

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Are you going to contribute something? Your description is too abstract to follow.

**Comment by gamlungJoeng** [comment]
When it comes to front-end bugs, developers usually want to fix them as quickly as possible. This is different from the server side (may add some task scheduling to document and improve the product). In web pages, js operation exceptions often appear on one or two users. Because each user has his own data and it is different from other users. So in addition to the call stack, when fixing these bugs you also need to send the wrong data (data is associated with the user and different for each user). I long for the SW UI to be able to see which user had the error when looking at the error stack.

**Reply by wu-sheng** [reply]
User is a kind of your application concept. SkyWalking can't tell

**Reply by wu-sheng** [reply]
https://github.com/apache/skywalking-client-js#according-to-different-pages-or-modules-add-custom-tags-to-spans

Tag is a thing we exposed for your custom data.

**Reply by wu-sheng** [reply]
If you want to change metric relative, you have to set custom endpoint to represent. But reporting all users metrics should be careful. You may face huge data set to crash the SkyWalking or database.

**Reply by gamlungJoeng** [reply]
Looks like this, I'll try something else
